src/plugins/docs/index.js

```diff
@@ -136,15 +89,46 @@ async function pluginDocs(context, options) {
     },
   };
 }
-pluginDocs.validateOptions = ({ validate, options }) => {
-  const { bannerPath, globalSidebars = [], ...docsOptions } = options;
+exports.default = pluginDocs;
+function validateOptions({ validate, options }) {
+  const { versions = {}, globalSidebars = [], ...docsOptions } = options;
+  const versionEntries = Object.entries(versions);
+  if (versionEntries.length > 1)
+    throw 'Multiple Docusuaurus doc versions not allowed in the Wiki';
+  // Handle version banner.
+  const versionBannerMap = {};
+  const docsVersionEntries = versionEntries.map(
+    ([versionLabel, versionOptions]) => {
+      // TODO: validate banner
```

Comment:
just in case, there's a todo here

Reply:
@jlvandenhout it is your todo. No clue what you wanted to do here 😆

Reply:
Normally you'd validate config in this function, but since the valid values of `banner` are hardcoded, it would fail with our additional values. So we take `banner` out of the options, before they are validated, and then re-add `banner` after valodation. I'm fine merging this as is.